cmd/sdn-cni-plugin/openshift-sdn_linux.go

```diff
@@ -126,9 +126,11 @@ var iptablesCommands = [][]string{
 	{"-A", "FORWARD", "-p", "tcp", "-m", "tcp", "--dport", "22623", "-j", "REJECT"},
 	{"-A", "FORWARD", "-p", "tcp", "-m", "tcp", "--dport", "22624", "-j", "REJECT"},
 
-	// Block cloud metadata IP
-	{"-A", "OUTPUT", "-d", "169.254.169.254", "-j", "REJECT"},
-	{"-A", "FORWARD", "-d", "169.254.169.254", "-j", "REJECT"},
+	// Block cloud metadata IP on all ports except 53 for tcp/udp
+	{"-p", "tcp", "-A", "OUTPUT", "-d", "169.254.169.254", "!", "--dport", "53", "-j", "REJECT"},
```

Comment:
It's convention to put the "-A OUTPUT" first.

Reply:
Thanks, just updated the PR